pkg/controllers/override/imagepatch.go

```diff
@@ -0,0 +1,180 @@
+/*
+Copyright 2023 The KubeAdmiral Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package override
+
+import (
+	"fmt"
+	"strings"
+
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+
+	fedcorev1a1 "github.com/kubewharf/kubeadmiral/pkg/apis/core/v1alpha1"
+	"github.com/kubewharf/kubeadmiral/pkg/controllers/common"
+	podutil "github.com/kubewharf/kubeadmiral/pkg/util/pod"
+)
+
+const (
+	pathSeparator   = "/"
+	OperatorAdd     = "add"
+	OperatorRemove  = "remove"
+	OperatorReplace = "replace"
+)
+
+var PodSpecPrefix = map[string]string{
```

Comment:
Can we reuse the definitions in https://github.com/gary-lgy/kubeadmiral/blob/60ac07e254541e7581da7c121079dcf08c77838b/pkg/util/pod/pod.go?

Reply:
The code has been updated.